apps/src/pythonlab/pythonHelpers/scripts.ts

```diff
@@ -0,0 +1,17 @@
+// Find all tests in the project and run them. The test files need to start with 'test'
+// and end with '.py'. The tests be in the top-level folder or in a folder with an
+// __init__.py file (the file can be empty). An init file signifies that the folder is a module.
+// In order to restrict to find tests in a specific folder or file we can parameterize this
+// and update the start_dir and/or pattern.
+export const FIND_AND_RUN_ALL_TESTS = `
+import unittest
+
+def discover_and_run(start_dir = '.', pattern = 'test*.py'):
+    loader = unittest.TestLoader()
+    test_suite = loader.discover(start_dir, pattern)
+    runner = unittest.TextTestRunner(verbosity=2)
+    result = runner.run(test_suite)
+    return result
```

Comment:
It seems that the way you have `asyncRun` set up propagates all meaningful errors from Python (e.g., not being able to find the `start_dir` or something) even though we don't have any explicit Python handling set up. 

If we wanted more user-friendly/customized output, presumably, we could eventually return something like:
```
return {
            'total': result.testsRun,
            'failures': len(result.failures),
            'errors': len(result.errors),
            'passed': result.testsRun - len(result.failures) - len(result.errors)
        }
   ```

And then parse it on the other side, but I think you had a follow-up ticket for that already!